src/proc/y16i-to-y16y16.cpp

```diff
@@ -0,0 +1,39 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include "y16i-to-y16y16.h"
+#include "stream.h"
+// CUDA TODO
+//#ifdef RS2_USE_CUDA
+//#include "cuda/cuda-conversion.cuh"
+//#endif
+
+namespace librealsense
+{
+    struct y16i_pixel { uint8_t rl : 8, rh : 4, ll : 8, lh : 4; int l() const { return lh << 8 | ll; } int r() const { return rh << 8 | rl; } };
+    void unpack_y16_y16_from_y16i_10(byte* const dest[], const byte* source, int width, int height, int actual_size)
+    {
+        auto count = width * height;
+// CUDA TODO
+//#ifdef RS2_USE_CUDA
+//        rscuda::split_frame_y16_y16_from_y12i_cuda(dest, count, reinterpret_cast<const y12i_pixel*>(source));
```

Comment:
Please change the comment function to `from_y16i`

Reply:
done